php7/Dockerfile

```diff
@@ -62,8 +64,9 @@ RUN curl -sL https://deb.nodesource.com/setup_6.x -o nodesource_setup.sh \
   && rm nodesource_setup.sh \
   && apt-get install -y nodejs
 
-RUN npm install -g gulp-cli \
-  && npm install -g gulp-sass
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - \
```

Comment:
We don't need to globally add gulp and sass. Just run `yarn run gulp` or `yarn run sass` to use the npm dependencies.